toolkit/Makefile

```
@@ -12,14 +12,17 @@ toolkit_root := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 SCRIPTS_DIR  ?= $(toolkit_root)/scripts
 
 # Build targets
+##help:var:CONFIG_FILE:<config_path>=Path to image configuration file to use.
```

```suggestion
##help:var:CONFIG_FILE:<config_path>=Path to image configuration file to use. Will add package dependencies and define final image.
```

Not sure I like my wording, but would be nice to call out its affects on the package scheduler specifically (the image build part is fairly self explanatory)